numpy/core/src/umath/umath_tests.c.src

```diff
@@ -227,7 +228,7 @@ static void
                     ptr_this += stride_d;
                     ptr_that += stride_d;
                 }
-                *(@typ@ *)data_out = @sqrt_func@(out);
+                *(@typ@ *)data_out = npy_@sqrt_func@(out);
```

Comment:
I have tried to figure out why this has not caused appveyour g before but have been unable to come up with a conclusion. This code has successfully compiled using regular Visual Studio and Visual C++ for Python 2.7 (both of which were done on Windows 2012 R2 and my local Windows 8.1).

It makes no sense because I can't find anything in my code that could possibly affect umath_tests.c, and it's not like the npymath library is affecting this :(